raiden/api/python.py

```diff
@@ -147,19 +147,27 @@ def leave_token_network(self, token_address, only_receiving=True):
         return connection_manager.leave(only_receiving)
 
     def get_connection_manager_funds(self, token_address):
-        """Get the connection manager for a specific token"""
-        connection_manager = self.raiden.connection_manager_for_token(token_address)
-        if not connection_manager:
-            return None
+        """Get the sum of all connection manager's open channels deposits"""
+        try:
+            connection_manager = self.raiden.connection_manager_for_token(token_address)
+        except InvalidAmount:
+            connection_manager = None
+        if connection_manager is not None and connection_manager.open_channels:
+            return connection_manager.sum_deposits
         else:
-            return connection_manager.funds
+            # explicitly return None, in case of invalid connection manager
```

Comment:
This `else` is much more than "invalid" connection manager. The way this is written it will return `None` even if there is a valid connection manager that currently has no open channels. For that case I believe `0` would be the correct result.

Reply:
From our discussion on chat, I did understand that a valid connection manager **is** one with open channels, as there's no way of differentiating it from a connection manager that was never joined (because `ConnectionManager` is always created for every token, not only those joined).